python/tvm/relay/op/nn.py

```diff
@@ -0,0 +1,54 @@
+"""Neural network operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+
+def conv2d(data,
+           weight,
+           strides=(1, 1),
+           padding=(0, 0),
+           dilation=(1, 1),
+           groups=1,
+           channels=None,
+           kernel_size=None,
+           data_layout="NCHW",
+           weight_layout="OIHW",
+           out_layout="",
+           out_dtype=""):
+    """Two dimensional convolution operator.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        The input data to the operator.
+
+    weight : relay.Expr
+        The weight expressions.
+
+    strides : tuple of int, optional
+        The strides of convoltution.
+
+    padding : tuple of int, optional
+        The padding of convolution on both sides of inputs.
+
+    dilation : tuple of int, optional
+        Specifies the dilation rate to be used for dilated convolution.
+
+    groups : int, optional
+        Number of groups for grouped convolution.
+
+    data_layout : str, optional
+        Layout of the input.
```

Comment:
Should we mention the possible options for `data_layout`, `weight_layout` etc?

Reply:
@junrushao1994 

It can be a sample then complete list if required.
The possible options list is very long and is explained in layout header.

Reply:
@srkreddy1238  My comment here is too picky. Definitely It is not a big deal.

I am just a little bit worried whether Python frontend users will be willing to check C APIs described in `include/tvm/relay/attrs/nn.h`.